### ISSUE TITLE
Integration update made Alexa not respond to voice now

### ISSUE THREAD

**Issue by Reachushere** [custom-components/alexa_media_player#2056]
**IMPORTANT: Please search the issues, including closed issues, and the [FAQ](https://github.com/custom-components/alexa_media_player/wiki/FAQ) before opening a new issue. The template is mandatory; failure to use it will result in issue closure.**

**Describe the bug**

 Alexa Media has had multiple upgrades lately.  The last upgrade makes Alexa not respond anymore.  I speak loudly, slowly, the light goes cyan as it’s listening, and no matter what command I use, it does nothing.  I have multiple echos, same problem on all of them.  Other people’s voice causes the same issue - Alexa doesn’t do any of the actions.  Alexa listens but does nothing, there is no response at all, no “sorry, I didn’t understand” or anything.  Examples are “Alexa turn off the lights” or “Alexa, stop all music.” among others.

IMPORTANT: Alexa is playing music as part of automations I have set up where the music turns on via the Spotify integration such as when I return home for example.  The echos play the music but then I can’t tell Alexa to turn the music off.  The cyan light comes on , she listens as I ask her to turn off all music, and then she doesn’t respond and the blue light goes off.

**To Reproduce**

**Expected behavior**

When asked, I expected Alexa to stop all music or Alexa would turn off the lights.  This issue only started happening when Alexa Media upgraded.

**Screenshots**

**System details**

- Home-assistant (version): 2023.9.3
- alexa_media (version from `const.py` or HA startup): I’m not sure what this is up I just updated the app yesterday or the day before, so it is very recent because the update was just released a few hours earlier
- alexapy (version from `pip show alexapy` or HA startup):
- Amazon [2FA is enabled](https://github.com/custom-components/alexa_media_player/wiki/Configuration#enable-two-factor-authentication-for-your-amazon-account) (y/n). : yes

**Logs**
Please provide [logs](https://github.com/custom-components/alexa_media_player/wiki/FAQ#how-do-i-enable-debug-logging-for-the-component).

I don’t understand how to do this.  I use the OS, but the instructions you have for how to do this say I am supposed to go to Configurator which doesn’t exist and then execute shell command but that doesn’t exist either???

**Additional context**

**Comment by alandtse** [comment]
Issues with the hardware aren't something this software can help with. Please consult with Amazon if your echo devices are non responsive.

**Comment by Reachushere** [comment]
They respond to the automations.  I have 22 echoes.  None of them are responding to voice, and only after the integration was installed.  How can they be working with the automations?On Oct 3, 2023, at 11:05 PM, Alan Tse ***@***.***> wrote:﻿
Issues with the hardware aren't something this software can help with. Please consult with Amazon if your echo devices are non responsive.
